RosNet/RosReader/UnParsedMessageHandler.cs

```diff
@@ -40,14 +40,18 @@ public void AddUnParsedMessage(Message message, byte[] data)
         /// </summary>
         public void ParseMessages(RosBag rosBag)
         {
-            foreach(KeyValuePair<int, Connection> kvp in rosBag.Connections)
+            int i = 0;
```

Comment:
??

Reply:
Forgot to remove that after debugging, will remove it